swift/apple/FirezoneNetworkExtension/NetworkSettings.swift

```
@@ -127,19 +127,25 @@ enum IPv4SubnetMaskLookup {
   ]
 }
 
-// Route convenience helpers. Data is from connlib and guaranteed to be valid.
-// Otherwise, we should crash and learn about it.
+// Route convenience helpers.
 extension NetworkSettings {
-  struct Cidr: Codable {
+  struct Cidr {
     let address: String
     let prefix: Int
 
-    var asNEIPv4Route: NEIPv4Route {
-      return NEIPv4Route(
-        destinationAddress: address, subnetMask: IPv4SubnetMaskLookup.table[prefix]!)
+    var asNEIPv4Route: NEIPv4Route? {
```

Instead of returning `nil`, I think it would be better to throw an error here.